index.html

```diff
@@ -1,20 +1,26 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
+	<meta charset="utf-8"/>
 	<title>Embedding images and videos</title>
 	<link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
 	<nav id="navbar">
 	<ul>
-		<li><a href="#"><img  src="menu.PNG"></a></li>
-		<li><a href="#"><img src="youtube.PNG"></a></li>
-		<li id="search"><input id="input" type="text" placeholder="Search" width="400px" margin-right="40px"><button id="search-button"><img id="button" src="search.PNG" ></button></li></a></li>
+		<li><a href="#"><img  src="img/menu.PNG" alt="Missing Img"></a></li>
```

Comment:
`alt` tag should have text related to the image. For example, if it's a cat image than `alt` should be a cat. It should contain information about that image.